src/Common/MoistThermodynamics/states.jl

```diff
@@ -113,6 +114,24 @@ function LiquidIcePotTempSHumEquil(θ_liq_ice::FT, q_tot::FT, ρ::FT) where {FT<
     return PhaseEquil(e_int, q_tot, ρ, T)
 end
 
+"""
+    LiquidIcePotTempSHumEquil_given_pressure(θ_liq_ice, q_tot, ρ)
+
+Constructs a [`PhaseEquil`](@ref) thermodynamic state from:
+
+ - `θ_liq_ice` - liquid-ice potential temperature
+ - `q_tot` - total specific humidity
+ - `p` - pressure
+"""
+function LiquidIcePotTempSHumEquil_given_pressure(θ_liq_ice::FT, q_tot::FT, p::FT) where {FT<:Real}
+    q = PhasePartition(q_tot)
+    Π_dry = exner_given_pressure(p)
+    T = Π_dry*θ_liq_ice + latent_heat_liq_ice(q)/cp_m(q)
```

Comment:
Can we make this a function that computes temperature from liquid-ice potential temperature given pressure?

Reply:
`air_temperature_from_liquid_ice_pottemp_given_pressure`? Yes, we can use that. I didn't use it initially because it didn't seem to match, exactly, what was [written down](https://github.com/climate-machine/CLIMA/pull/508#discussion_r342677797).